homeassistant/components/ombi/sensor.py

```diff
@@ -0,0 +1,125 @@
+"""Support for Ombi."""
+from datetime import timedelta
+import logging
+
+import pyombi
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_HOST,
+    CONF_MONITORED_CONDITIONS,
+    CONF_PORT,
+    CONF_SCAN_INTERVAL,
+    CONF_SSL,
+)
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_URLBASE = "urlbase"
+
+DEFAULT_PORT = 5000
+DEFAULT_SCAN_INTERVAL = timedelta(seconds=60)
+DEFAULT_SSL = False
+DEFAULT_URLBASE = ""
+
+SENSOR_TYPES = {
+    "movies": {"type": "Movie requests", "icon": "mdi:movie"},
+    "tv": {"type": "TV show requests", "icon": "mdi:television-classic"},
+    "pending": {"type": "Pending requests", "icon": "mdi:clock-alert-outline"},
+    "approved": {"type": "Approved requests", "icon": "mdi:check"},
+    "available": {"type": "Available requests", "icon": "mdi:download"},
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_API_KEY): cv.string,
+        vol.Required(CONF_HOST): cv.string,
+        vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+        vol.Optional(CONF_URLBASE, default=DEFAULT_URLBASE): cv.string,
+        vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
+        vol.Optional(CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL): cv.time_period,
```

Comment:
Please define the `SCAN_INTERVAL` constant at the module level instead. This will set the default scan interval of this platform.

Reply:
Should be fixed now.